UserDataProcessingSaveFailureReasonActivity/handler.ts

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
Is this activity actually used? Also, whoudn't `SetUserDataProcessingStatusActivity` be sufficient?

Reply:
No, this is not used actually. In the end I decided to use SetUserDataProcessingStatusActivity and forgot to delete this.